README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+
+---
+#"Computing for Social Sciences HW01"
+"**Taylor Griffin**"  
+"*October 8, 2019*"  
+
+# Introduction
+
+![Taylor Griffin](C:/Users/Taylor Griffin/Desktop/Computing R/Taylor G hw01/Taylor.png)
```

Comment:
Your picture is not showing because you are using an absolute file path from your operating system. We will cover the difference between a relative path/absolute file path later in class.